### ISSUE TITLE
Dashed vs. dotted lines in output

### ISSUE THREAD

**Issue by rduesing** [melissagwolf/dynamic#23]
I am a bit confused by the dotted and dashed lines in the plots, se attached file.

![dfi_plot](https://github.com/user-attachments/assets/368258d0-f8d5-41f0-ad95-ccb71a6d5cde)

Shouldn't the caption be the other way around? Isn't the dotted line the Hu & Bentler cut-off and the dashed line the dynamic value? Am I missing something?

**Comment by melissawolf-delfi** [comment]
Yes, you're right. `ggplot` seems to switch this on us once a year, or so. I'm not sure why.
